src/main/scala/tech/cryptonomic/conseil/util/FPUtil.scala

```diff
@@ -0,0 +1,11 @@
+package tech.cryptonomic.conseil.util
+
+import scala.util.{Success, Try}
+
+object FPUtil {
+
+  def sequence[A](a: List[Try[A]]): Try[List[A]] = {
```

Comment:
implementation is correct

if you want to get a bit further and have both `traverse` and `sequence`, we can partly reuse the implementation from cats

```scala
def map2[A, B, Z](ta: Try[A], tb: Try[B])(f: (A, B) => Z): Try[Z] =
  (ta, tb) match {
    case (Success(a), Success(b)) => Try(f(a,b))
    case (f: Failure[_], _)       => f.asInstanceOf[Try[Z]]
    case (_, f: Failure[_])       => f.asInstanceOf[Try[Z]]
  }

def traverse[A, B](ts: List[A])(f: A => Try[B]): Try[List[B]] =
  ts.foldRight[Try[List[B]]](Success(List.empty)) {
    (a, tbs) => map2(f(a), tbs)(_ :: _)
  }

def sequence[A](ts: List[Try[A]]): Try[List[A]] = traverse(ts)(identity)
```

and be done with it

Reply:
Awesome, thanks!